srcpkgs/nvidia470/template

```diff
@@ -0,0 +1,311 @@
+# Template file for 'nvidia470'
+
+_desc="nvidia470 drivers for linux"
```

Comment:
To match `nvidia390`, maybe:
 
    _desc="NVIDIA drivers (GeForce 600, most 700 series)"